mkdocs.yml

```diff
@@ -34,28 +35,33 @@ nav:
 
   - Home: index.md
   - Getting Started: getting_started.md
-  - Bill of Materials: bill_of_materials.md
-  - Hardware Setup: hardware_setup.md
-  - Software Setup: 
-      - Using SD Card Image: software_setup/sd_card.md
+  - Hardware:
```

Comment:
Given the hardware/software setup are dependent (ie: third party motor driver may require alternate HW setup).  We may want to avoid dividing along the hardware/software line.  The following would work...

- Home
- Getting Started
- Platforms
   - DIY JetBot
      - Bill of Materials
      - Hardware Setup
      - Software Setup
          - Using Docker Container
          - Using SD Card Image
   - Third Party Kits
- Examples
- Reference

However, this involves a lot of nesting.  Fortunately, since the docs are directly associated with our GitHub project (which is the owning repository of the "DIY" jetbot stuff")  I think the following will feel natural to visitors.


- Home
- Getting Started
- Bill of Materials
- Hardware Setup
- Software Setup
  - Using Docker Container
  - Using SD Card Image
- Examples
- Reference
   - Third Party Kits

The getting started guide lists "Third Party Kits" as a standard option now, so I think it is marketed sufficiently.

I feel either of these options addresses the structural issues.  I personally prefer option (2) with the current state of things.